src/lib/public/board/data/officers.json

```diff
@@ -1,718 +1,368 @@
-      },
-      "F22": {
-        "title": "Treasurer",
-        "tier": 4
-      }
-    },
-    "picture": "jacob-nguyen.webp"
+      "S21": { "title": "President, Create Director, nodebuds Officer", "tier": 0 },
+      "F22": { "title": "Treasurer", "tier": 4 }
+    }
   },
   {
     "fullName": "Taylor Noh",
+    "picture": "taylor-noh.webp",
     "positions": {
-      "S21": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      },
-      "F21": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      },
-      "S22": {
-        "title": "nodebuds Officer",
-        "tier": 23
-      }
-    },
-    "picture": "taylor-noh.webp"
+      "S21": { "title": "nodebuds Officer", "tier": 22 },
+      "F21": { "title": "nodebuds Officer", "tier": 22 },
+      "S22": { "title": "nodebuds Officer", "tier": 22 }
+    }
   },
   {
     "fullName": "Kirsten Ochoa",
+    "picture": "kirsten-ochoa.webp",
     "positions": {
-      "S22": {
-        "title": "Create Project Developer",
-        "tier": 18
-      },
-      "F22": {
-        "title": "Design Officer",
-        "tier": 18
-      }
-    },
-    "picture": "kirsten-ochoa.webp"
+      "S22": { "title": "Create Project Developer", "tier": 14 },
+      "F22": { "title": "Design Officer", "tier": 16 }
+    }
   },
   {
     "fullName": "Mike Ploythai",
+    "picture": "mike-ploythai.webp",
     "positions": {
-      "S21": {
-        "title": "Create Officer",
-        "tier": 17
-      },
-      "F21": {
-        "title": "Create Director, Marketing Chair",
-        "tier": 16
-      },
-      "S22": {
-        "title": "Create President",
-        "tier": 16
-      }
-    },
-    "picture": "mike-ploythai.webp"
+      "S21": { "title": "Create Officer", "tier": 16 },
+      "F21": { "title": "Create Director, Marketing Chair", "tier": 13 },
+      "S22": { "title": "Create President", "tier": 13 }
+    }
   },
   {
     "fullName": "Stephanie Pocci",
+    "picture": "stephanie-pocci.webp",
     "positions": {
-      "S22": {
-        "title": "Event Coordinator",
-        "tier": 9
-      },
-      "F22": {
-        "title": "Event Coordinator",
-        "tier": 9
-      }
-    },
-    "picture": "stephanie-pocci.webp"
+      "S22": { "title": "Event Coordinator", "tier": 10 },
+      "F22": { "title": "Event Coordinator", "tier": 10 }
+    }
   },
   {
     "fullName": "Nicolas Renteria",
-    "positions": {
-      "S21": {
-        "title": "Marketing Chair",
-        "tier": 12
-      }
-    },
-    "picture": "nicolas-renteria.webp"
+    "picture": "nicolas-renteria.webp",
+    "positions": { "S21": { "title": "Marketing Chair", "tier": 8 } }
   },
   {
     "fullName": "Wilbert Rodriguez",
-    "positions": {
-      "S21": {
-        "title": "Intern Program Manager",
-        "tier": 11
-      }
-    },
-    "picture": "wilbert-rodriguez.webp"
+    "picture": "wilbert-rodriguez.webp",
+    "positions": { "S21": { "title": "Intern Program Manager", "tier": 22 } }
```

Comment:
I suggest to actually switch the tier from "22" to "23". 

`"Nodebuds Officer": { "id": 22, "index": 1200 },`
`"ICC Representative": { "id": 23, "index": 1250 }`